### PR TITLE
hbase-store: kv test

### PR DESCRIPTION
**Pull request by disksing**
Run `go test -teststore hbase -testpath zk1,zk2,zk3/testkv` to test hbase storage.

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

File: store/localstore/mvcc_test.go [hunk 1]
```diff
@@ -301,3 +301,13 @@ func (t *testMvccSuite) TestBufferedIterator(c *C) {
 	c.Assert(it.Value(), DeepEquals, []byte("2"))
 	tx.Commit()
 }
+
+func encodeInt(n int) []byte {
```
Comment (qiuyesuifeng) [inline store/localstore/mvcc_test.go]: These two functions seem not be used?

Comment (disksing) [inline store/localstore/mvcc_test.go]: These functions was in `kv_test.go` and used by both `kv_test.go` and "mvcc_test.go" before. As `localsotre/kv_test.go` was moved to `store/store_test.go` in this commit, we need add them back to `mvcc_test.go`.

**Comment by qiuyesuifeng** [comment]
LGTM

**Pull request merged**